pom.xml

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
-
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
     <version>3.10</version>
     <relativePath />
   </parent>
-  <groupId>io.jenkins</groupId>
-  <artifactId>configuration-as-code</artifactId>
+  <groupId>io.jenkins.configuration-as-code</groupId>
```

Comment:
maybe we should also add "plugins" to groupId

Reply:
use of io.jenkins was by intent, as we expected this project to become a top level one (like remoting is).